x-pack/plugin/core/src/main/java/org/elasticsearch/protocol/xpack/graph/GraphExploreRequest.java

```diff
@@ -187,12 +163,16 @@ public GraphExploreRequest timeout(String timeout) {
         return this;
     }
 
+    private static final String[] EMPTY_ARRAY = new String[]{};
```

Comment:
Instead of defining this variable, I think we could use `Strings.EMPTY_ARRAY` below.

Reply:
I knew we had a predefined array somewhere, and for some reason was unable to find it. Thanks!